docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -233,9 +232,7 @@ intervention. Scales linearly over 1 year such that there is 0% coverage at base
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 * :ref:`Systolic Blood Pressure <2019_risk_effect_sbp>`
 * :ref:`High LDL Cholesterol <2019_risk_effect_ldl>`
-
-.. todo::
-  Create risk effect models for BMI and FPG 
```

Comment:
Don't you still want the fpg to-do?

Reply:
Adding FPG is in a PR now, so its almost done!